fusillade-api.yml

```diff
@@ -1122,13 +1359,19 @@ components:
       type: array
       items:
         type: string
+    policy_type:
+      type: string
+      enum: ['ResourcePolicy', 'IAMPolicy']
```

Comment:
Is the IAMPolicy type TBA? Would it be better to modify this to `['ResourcePolicy']` only until IAMPolicy policies are added?

Reply:
It's already used in other places in the code, such as roles, users, and groups. A resource can potential have IAM policies attached to it as well. I'll remove it until it's used.